staging/Vagrantfile

```diff
@@ -0,0 +1,44 @@
+Vagrant.require_version ">= 1.8.2"
+
+domain = 'localdomain'
+box = "bento/ubuntu-18.04"
+ram_size_mb = '16384'
+
+# vault_config_file = 'shared/vault_config'
```

Comment:
These commented lines are obsolete?

Reply:
yeah - I was thinking we'd put them back in soon but am not currently that sure that they're needed...